Implement Metros and update the default OS, plan, and location

displague opened the pull request:
Introduces metros and updates the defaults to those available to all users.

Fixes #60 
Fixes #61

displague committed:
update packngo to 0.13.0

Signed-off-by: Marques Johansson <mjohansson@equinix.com>

displague committed:
add metro support and update default OS

Signed-off-by: Marques Johansson <mjohansson@equinix.com>

displague committed:
default metro only when facility is not provided

Signed-off-by: Marques Johansson <mjohansson@equinix.com>

jmarhee commented on pkg/drivers/metal/metal.go:
```diff
@@ -22,6 +23,8 @@ import (
 	dockerConfigDir = "/etc/docker"
 	consumerToken   = "24e70949af5ecd17fe8e867b335fc88e7de8bd4ad617c0403d8769a376ddea72"
+	defaultOS       = "ubuntu_20_04"
+	defaultMetro    = "DC"
```
The metro should be lower-cased:

```
$ docker-machine create sloth --driver metal --metal-api-key=$API_KEY --metal-project-id=$PROJECT --metal-plan "baremetal_0" --metal-metro-code "DC"
Running pre-create checks...
Error with pre-create check: "metal requires a valid metro"
```
Runs as expected with `"dc"` from the CLI

displague replied:
I intended to change the default plan from baremetal_0 to c3.small.x86 so that the defaults would accommodate more users (new users).

jmarhee reviewed (changes requested):
Just that one note, but otherwise, ran as expected.

displague committed:
fix defaultMetro (use "dc" not "DC")

Signed-off-by: Marques Johansson <mjohansson@equinix.com>

displague committed:
change default plan from baremetal_0 to c3.small.x86

baremetal_0 is an alias for t1.small.x86 which is not available to new
Equinix Metal users. c3.small.x86 is the smallest plan available to all
users.

Signed-off-by: Marques Johansson <mjohansson@equinix.com>

jmarhee reviewed (approved):
Looks great!
